packages/p2p-v2/src/components/Table/Table.tsx

```diff
@@ -33,23 +34,49 @@ const Table = <T,>({
     });
 
     const tableContainerRef = useRef<HTMLDivElement>(null);
+    const headerRef = useRef<HTMLDivElement>(null);
+    const [height, setHeight] = useState(0);
+
+    useLayoutEffect(() => {
+        if (headerRef?.current) {
+            const topPosition = headerRef.current.getBoundingClientRect().bottom;
+            setHeight(window.innerHeight - topPosition);
+        }
+    }, [headerRef?.current]);
+
     const { fetchMoreOnBottomReached } = useFetchMore({
         loadMore: loadMoreFunction,
         ref: tableContainerRef,
         isFetching,
     });
 
     return (
-        <div
-            className={clsx('p2p-v2-table', tableClassname)}
-            onScroll={e => fetchMoreOnBottomReached(e.target as HTMLDivElement)}
-            ref={tableContainerRef}
-        >
-            {table.getRowModel().rows.map(row => (
-                <div className={rowClassname} key={row.id}>
-                    {rowRender(row.original)}
+        <div className='w-full'>
+            {isDesktop && columns.length > 0 && (
+                <div
+                    className='p2p-v2-table__header'
```

Comment:
nit: later we need to install `twMerge`, we could do stuff liek:
```
import { ClassValue, clsx } from 'clsx'
import { twMerge } from 'tailwind-merge'

export const cn = (...inputs: ClassValue[]) => {
  return twMerge(clsx(inputs))
}
```
andd
```
cn('p2p-v2-table__header', `grid-cols-${columns.length}`)
```

Reply:
so in the future if we wanna inline stuff we just tailwind and merge it

Reply:
yeah.. but tbh i didn't want to do much inside this table since it will be replaced by the derivcom one..